src/main/java/frc/robot/subsystems/Shooter.java

```diff
@@ -70,14 +70,18 @@ public CommandBase dump() {
      * Calculates RPM with some gear ratio mathematics. (returns inches/second) Also
      * applies a 10% loss.
      */
-    public static double calculateRPM() {
+    public CommandBase CalculatedShoot() {
+        final double RPM_SPEED;
         if (SmartDashboard.getBoolean("Sees Target", false)) {
-            return SmartDashboard.getNumber("Last RPM", 0);
+            RPM_SPEED = SmartDashboard.getNumber("Last RPM", 0);
```

Comment:
Add some comments explaining the logic - why does it calculate if it doesn't see a target? If it sees a target it just uses the last calculation?